pkg/kube/multicluster/secretcontroller.go

```diff
@@ -91,15 +92,36 @@ type Controller struct {
 
 // NewController returns a new secret controller
 func NewController(kubeclientset kube.Client, namespace string, localClusterID cluster.ID) *Controller {
+	// When these two are set to true, Istiod will be watching the namespace in which
+	// Istiod is running on the external cluster. Use the inCluster credentials to
+	// create a kubeclientset
+	informerClient := kubeclientset
```

Comment:
Not sure if creating k8s clients all over the place is a good pattern - I would guess at some point we may want to load other things from the 'local cluster'. 

Would it be possible to init the local cluster where we initialize config and remote clusters, and keep it in a field ?